pingpong/ud_pingpong.c

```diff
@@ -449,16 +449,26 @@ static int server_connect(void)
 {
 	int ret;
 	struct fi_cq_entry comp;
+	struct timespec a, b;
 
 	ret = common_setup();
 	if (ret != 0)
 		goto err;
 
+	clock_gettime(CLOCK_REALTIME_COARSE, &a);
```

Comment:
FWIW, the same issue occurs in the recv_xfer() routine (i.e., commit b79d536).  I didn't notice any difference in the reported times for short message latency, even with the unconditional clock_gettime().